luigi/contrib/spark.py

```diff
@@ -122,6 +122,10 @@ def jars(self):
         return self._list_config(configuration.get_config().get("spark", "jars", None))
 
     @property
+    def packages(self):
+        return self._list_config(configuration.get_config().get("spark", "packages", None))
```

Comment:
yeah but this complexity has been there for a long time in the spark class, so we should fix it in a separate PR